common.blocks/dropdown/dropdown.js

```diff
@@ -31,8 +31,20 @@ provide(BEMDOM.decl(this.name, /** @lends dropdown.prototype */{
             }
         },
 
-        'opened' : function(_, modVal) {
-            this.getPopup().setMod('visible', modVal);
+        'opened' : {
+            '*' : function(_, modVal) {
+               this.getPopup().setMod('visible', modVal);
+               this._switcher.domElem.attr('aria-expanded', !!modVal);
+           },
+
+           'true' : function() {
+               this._popup.domElem.attr('tabindex', 0).focus();
```

Comment:
This solution contradicts our a11y manual: https://nda.ya.ru/3Quhd7

>Focus should be on button after dropdown expansion.

Let's solve focus maintaining problems later (#1026) and separately from readability.